wristband/apps/providers.py

```diff
@@ -1,9 +1,12 @@
+import datetime
 import requests
 from django.conf import settings
 
-from wristband.common.utils import extract_stage
+from wristband.common.utils import extract_stage, extract_security_zone_from_env
+from wristband.providers.generics import JsonDataProvider
+from wristband.providers.models import Job
 
-from wristband.common.providers import JsonDataProvider
+EXPIRY_JOB_TIME = datetime.datetime.now() - datetime.timedelta(minutes=10)
```

Comment:
Won't this be set to now - 10 minutes at import time and then never changed for the life of the program? My bad if that's what you're going for.